ai8x.py

```diff
@@ -1473,7 +1473,7 @@ def _fuse_bn_layers(m):
 
                 r_mean = target_attr.bn.running_mean
                 r_var = target_attr.bn.running_var
-                r_std = torch.sqrt(r_var + 1e-20)
+                r_std = torch.rsqrt(r_var + target_attr.bn.eps)
```

Comment:
It looks good.

Not an important issue but is there a specific reason why we use reciprocal sqrt here? Should we rename the variable to r_inv_std for clarity?

Reply:
reciprocal sqrt is used here as it is safer to handle division by very small numbers especially when gradient calculation is important. The name of the variable is changed as you suggest.

Reply:
Thanks!